extensions/inline/italic.rng

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Having a second look, and maybe it is more of a topic of discussion than a review point: I wonder if the name ``/inline/`` is borrowing a bit from HTML. For example, it looks like in JATS ``<italic>`` is part of Emphasis tags. Would you also have to start defining each tag as inline or alternate (is there only one non-inline type, or multiple non-inline ways to group tags?). 

As an alternative, how would it looks with folders like ``extensions/italic/italic.rng``, and then ``extensions/italic/full/basic.rng``, etc. ?